.github/workflows/trigger-preview.yaml

```
@@ -0,0 +1,28 @@
+name: trigger-preview
+on:
+  workflow_run:
+    workflows:
+      - trigger-book-build
+    types:
+      - requested
+      - completed
+
+jobs:
+  find-pull-request:
+    uses: ProjectPythiaCookbooks/cookbook-actions/.github/workflows/find-pull-request.yaml@main
+  deploy-preview:
+    needs: find-pull-request
+    if: github.event.workflow_run.conclusion == 'success'
+    uses: ProjectPythiaCookbooks/cookbook-actions/.github/workflows/deploy-book.yaml@main
+    with:
+      artifact_name: book-zip-${{ needs.find-pull-request.outputs.number }}
+      destination_dir: _preview/${{ needs.find-pull-request.outputs.number }}  # deploy to subdirectory labeled with PR number
```

```suggestion
      destination_dir: _preview/${{ needs.find-pull-request.outputs.number }} # deploy to subdirectory labeled with PR number
```

The necessary change to satisfy prettier. I'll give this a full review later as able!

---

Thanks, that did the trick.